open-craft/sprints
# [BB-1246] End sprints with a button and upload spillovers to Google Spreadsheets

This adds a uploading spillovers to Google Spreadsheets, ending sprints and activating future ones.
**Note**: Currently this PR's target branch is `bb-1245-vacation` for allowing parallel reviews. I'll switch this to `master` after merging that branch.

## Testing instructions:
1. Set `.env` according to the ticket description.
1. Start the backend with `docker-compose -f local.yml up --build`.
1. Create superuser with `docker-compose -f local.yml run --rm django python manage.py createsuperuser` (the button on frontend should be visible for `staff` users only).
1. Create `frontend/.env.local` and set `REACT_APP_GOOGLE_CLIENT_ID` there.
1. Navigate to `frontend` and run `npm i && npm start`.
1. Log in.
1. Click the red button.
1. Check the spreadsheet for the spillover tickets.
